conga-aem-definitions/src/main/roles/aem-dispatcher.yaml

```diff
@@ -159,6 +170,7 @@ files:
   variants:
   - aem-publish*
   - ssl*
+  - http2*
```

Comment:
this is wrong - this would generate the whole config file only if all of aem-publish, ssl and http2 variants are enabled. just remove this addition, should work without.